src/main/java/org/spongepowered/api/world/generation/config/noise/Shaper.java

```diff
@@ -22,35 +22,52 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.api.world.biome.provider.multinoise;
+package org.spongepowered.api.world.generation.config.noise;
 
 import org.spongepowered.api.Sponge;
 
-import java.util.List;
-import java.util.Objects;
+public interface Shaper {
 
-public interface MultiNoiseConfig {
+    static Shaper overworld() {
+        return Sponge.game().factoryProvider().provide(Shaper.Factory.class).overworld(false);
+    }
+
+    static Shaper amplifiedOverworld() {
+        return Sponge.game().factoryProvider().provide(Shaper.Factory.class).overworld(true);
```

Comment:
Why does toggling overworld true mean amplified? If so, rename this, this is very confusing.